tests/system/dcos_service_marathon_tests.py

```diff
@@ -11,6 +11,14 @@
 from dcos.errors import DCOSUnprocessableException
 
 
+@retrying.retry(wait_fixed=1000, stop_max_attempt_number=16, retry_on_exception=common.ignore_exception)
+def assert_in_deployment(deployment_id):
```

Comment:
The name is misleading - the method checks `readinessCheckResults` for a given deployment so wdyt about `assert_deployment_not_ready` or smth.?

Reply:
✅